src/networking.c

```diff
@@ -2162,9 +2178,10 @@ sds catClientInfoString(sds s, client *client) {
         total_mem += zmalloc_size(client->argv);
 
     return sdscatfmt(s,
-        "id=%U addr=%s %s name=%s age=%I idle=%I flags=%s db=%i sub=%i psub=%i multi=%i qbuf=%U qbuf-free=%U argv-mem=%U obl=%U oll=%U omem=%U tot-mem=%U events=%s cmd=%s user=%s",
+        "id=%U addr=%s target=%s %s name=%s age=%I idle=%I flags=%s db=%i sub=%i psub=%i multi=%i qbuf=%U qbuf-free=%U argv-mem=%U obl=%U oll=%U omem=%U tot-mem=%U events=%s cmd=%s user=%s",
```

Comment:
I agree with @oranagra that `target` is a bit obscure; `bind` is better, also `laddr` is an option IMHO.

Reply:
what's the 'l' for in `laddr`?

Reply:
i'm guessing "local", but since i'd have to guess (unless it's a common thing), maybe not a good idea either.
i suppose it must be a combination of two words "local", "bind" or "target" with "addr".

i'm not sure for what reason this dict is really trying to shoose short names (like "obl" and "oll"). if there is a good reason, then "laddr", "baddr" or "taddr" may be ok (documentation will explain what it is).
but if there isn't a good reason for that, i'd go with "bind-addr", or "local-addr".

p.s. will the local address always be the bind address (unless bound on *)?

Reply:
@oranagra I prefer to stick to convention. While `laddr` (listen / local addr) maybe less explicit I think it's at least as easy to understand as `obl` or `oll`. I'm OK with it.

Yes you can expect the local address to be the bind address but I wouldn't count on it in a way that breaks something, one must always be prepared for socket layer surprises.

Reply:
done.